### PR TITLE
Added social network links from data dynamically

### LINKED ISSUE #6: Add social networks links to the data
Right now the CardComponent has fixed icons for a website, LinkedIn and Twitter, we need to link those values from the developers.json and only show them if they exists.

### PR DESCRIPTION
**Pull request by JeanCCV**
Fixes #6

### CODE REVIEW

File: DominicanWhoCodes.Blazor/Pages/Index.razor [hunk 1]
```diff
@@ -33,5 +34,8 @@
         public string Image { get; set; }
         public string Summary { get; set; }
         public string Skills { get; set; }
+        public string Wegpage { get; set; }
```
Comment (robertlluberes) [inline DominicanWhoCodes.Blazor/Pages/Index.razor]: Hey Jean, this should be `Webpage`.

File: DominicanWhoCodes.Blazor/Pages/Index.razor [hunk 2]
```diff
@@ -33,5 +34,8 @@
...
+        public string Linkedin { get; set; }
```
Comment (robertlluberes) [inline DominicanWhoCodes.Blazor/Pages/Index.razor]: This is a compose work, should be `LinkedIn`

**Comment by AngelGarcia13** [comment]
@JeanCCV Please check the comments from @robertlluberes .... Thanks for the contribution!

File: DominicanWhoCodes.Blazor/Pages/Index.razor (hunk 1 above)
Comment (JeanCCV) [reply DominicanWhoCodes.Blazor/Pages/Index.razor]: Thanks for the observation

File: DominicanWhoCodes.Blazor/Pages/Index.razor (hunk 2 above)
Comment (JeanCCV) [reply DominicanWhoCodes.Blazor/Pages/Index.razor]: I need a nacho book

**Pull request merged**